src/placement/tests/jump_map_place_obj.c

```diff
@@ -535,19 +536,19 @@ jtc_get_layout_rebuild_count(struct jm_test_ctx *ctx)
 static bool
 jtc_layout_has_duplicate(struct jm_test_ctx *ctx)
 {
-	int i;
-	int target_num;
-	bool *target_set;
-	bool result = false;
+	int      target_num;
+	bool    *target_set;
+	bool     result = false;
+	uint32_t total_targets;
 
 	D_ASSERT(ctx != NULL);
 	D_ASSERT(ctx->po_map != NULL);
-	const uint32_t total_targets = pool_map_target_nr(ctx->po_map);
+	total_targets = pool_map_target_nr(ctx->po_map);
 
 	D_ALLOC_ARRAY(target_set, total_targets);
 	D_ASSERT(target_set != NULL);
 
-	for (i = 0; i < ctx->layout->ol_nr; i++) {
+	for (int i = 0; i < ctx->layout->ol_nr; i++) {
```

Comment:
i guess this is going the other way now?

Reply:
I'm backing out this one, there's no other changes needed in this function.